aplicaciones/academico/views.py

```
@@ -8,18 +10,20 @@
     MateriaCursoSerializer,
     CreateGradoSerializer,
     CreateMateriaCursoSerializer,
+    ClaseSerializer
 )
 
 from aplicaciones.usuarios.utils import registrar_bitacora
 from aplicaciones.usuarios.usuario_views import get_client_ip 
 from rest_framework import viewsets, status
```

Ruff (0.11.9)

Redefinition of unused `viewsets` from line 3

Remove definition: `viewsets`

(F811)